client/src/components/Menu/index.tsx

```diff
@@ -149,6 +159,23 @@ class Menu extends React.Component<IMenuProps> {
                                                 </DropdownItem>
                                                 <DropdownItem divider/>
                                             </div>
+                                            {sharp === searchValue && search_hashtags.map((hashtag: any) => (
```

Comment:
do u need this const? search === 'e' isn't better?

Reply:
Fixed